src/main/java/http_server/Router.java

```diff
@@ -94,7 +82,7 @@ public void delete(String route, Callback handler) {
     }
 
     public void all(String route, Callback handler) {
```

Comment:
I can't find where this is actually used

Reply:
I use it for redirects